CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # master
 
+# v2.1.0
+
+* Support rendering collection (e.g., `render(MyComponent.all(@items))`).
```

Comment:
Just dropping a note here to make sure we update this to reflect whatever API we land on ❤️